noaa/operations/default.json

```diff
@@ -0,0 +1,112 @@
+    {
+      "name": "index",
+      "operation-type": "index",
+      "bulk-size": 5000
+    },
+    {
+      "name": "range_query_range_field",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "range": {
+            "TRANGE": {
+              "gte": -10,
+              "lte": 10
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "range_query_range_field_in_conjunction_with_term_query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+                "term": {
+                  "station.keyword": "ASN00001018"
+                }
+              },
+              {
+                "range": {
+                  "TRANGE": {
+                    "gte": -10,
+                    "lte": 10
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "name": "range_query_range_field_in_conjunction_with_terms_query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+                "terms": {
+                  "station.keyword": [
+                    "ASN00001018",
+                    "AQW00061705",
+                    "ARM00087453",
+                    "ARM00087576",
+                    "ASN00001006",
+                    "ASN00001007",
+                    "ASN00001010",
+                    "ASN00001013",
+                    "ASN00001018",
+                    "ASN00001019",
+                    "ARM00087121",
+                    "ASN00004032",
+                    "ASN00004046",
+                    "ASN00004019",
+                    "ASN00003105",
+                    "ASN00003100",
+                    "ASN00004083"
+                  ]
```

Comment:
I'm a bit worried that the overhead of merging postings of multiple terms will add noise. Maybe we could cross this dataset with stations (ftp://ftp.ncdc.noaa.gov/pub/data/ghcn/daily/ghcnd-stations.txt) in order to be able to index more metadata with all documents such as geo coordinates, state and elevation of the station. Then I believe we could find some states that have significant numbers of records?